dartagnan/src/main/java/com/dat3m/dartagnan/program/analysis/alias/AndersenAliasAnalysis.java

```diff
@@ -206,14 +206,14 @@ private void processResults(Local e) {
             addTarget(reg, new Location(mem, 0));
             return;
         }
-        if (!(exp instanceof IExprBin iBin)) {
+        if (!(exp instanceof IntBinaryExpr iBin)) {
             return;
         }
         Expression base = iBin.getLHS();
         if (base instanceof MemoryObject mem) {
             Expression rhs = iBin.getRHS();
-            //FIXME Address extends IConst
-            if (rhs instanceof IConst ic) {
+            //FIXME Address extends IntLiteral
```

Comment:
I don't get what these FIXMEs mean. Anything we can do about them?

Reply:
We can remove it. This was a very old FIXME where we still had the `Address` class that extended `IConst`.
IIRC, this let to problems because `Addr(5) + 8` would get simplified to `5 + 8 = 13`, i.e., it would treat the address as a regular int, do constant folding and thereby killing the address operand.